includes/html/graphs/peering/bits.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use LibreNMS\Config;
+
+// Generate a list of ports and then call the multi_bits grapher to generate from the list
+
+$cust_descrs = (array) Config::get('peering_descr', ['peering']);
+
+$sql = 'SELECT * FROM `ports` AS I, `devices` AS D WHERE `port_descr_descr` = ? AND D.device_id = I.device_id AND `port_descr_type` IN ' . dbGenPlaceholders(count($cust_descrs));
+$param = $cust_descrs;
+array_unshift($param, $vars['id']);
+
+$rrd_list = [];
+foreach (dbFetchRows($sql, $param) as $port) {
```

Comment:
you only need to convert this query to eloquent

Reply:
Allright, fixed.